src/components/SnackChat/SnackChat.js

```diff
@@ -156,11 +246,12 @@ class SnackChat extends Component {
 
     // Video background
     this.ctx.save();
-    this.ctx.scale(-1, 1);
+    this.ctx.scale(-1.6, 1.6);
```

Comment:
these 1.6 seem to be flighing around a lot
variable for these too if you think?

Reply:
you have video background up the top
do you need both? function?